ci/test.sh

```diff
@@ -113,7 +113,14 @@ test_template() {
     popd
 }
 
-for x in $assets_dir/*.yaml
+if [ "$TRAVIS" == "true" ]
```

Comment:
This is not a TRAVIS-only path.

Reply:
Even the travis build can (and should) use the -local index, as it has the path to the _built_ tarballs, rather than to those that haven't been published yet.